<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Search Demo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="categoryBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Category</string>
    </property>
    <widget class="QRadioButton" name="bagRadioButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>bag</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">categoryGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="shoeRadioButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>shoe</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">categoryGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="womanShoeRadioButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>101</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>woman_shoe</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">categoryGroup</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Query Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showAllButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Show All</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Query Type</string>
    </property>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>color</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">queryTypeGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>shape</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">queryTypeGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="radioButton_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>combined</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">queryTypeGroup</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>600</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>831</width>
      <height>531</height>
     </rect>
    </property>
    <property name="title">
     <string>Query Results</string>
    </property>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>500</y>
       <width>101</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>show ranks</string>
     </property>
    </widget>
    <widget class="QLabel" name="queryResult_1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>151</width>
       <height>151</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Images similar to this will be retrieved</string>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="midLineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="queryResult_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>151</width>
       <height>151</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Images similar to this will be retrieved</string>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="midLineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="queryResult_3">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>20</y>
       <width>151</width>
       <height>151</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Images similar to this will be retrieved</string>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="midLineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="queryResult_4">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>20</y>
       <width>151</width>
       <height>151</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Images similar to this will be retrieved</string>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="midLineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="queryResult_5">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>20</y>
       <width>151</width>
       <height>151</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Images similar to this will be retrieved</string>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="midLineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="queryResult_7">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>180</y>
       <width>151</width>
       <height>151</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Images similar to this will be retrieved</string>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="midLineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="queryResult_8">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>180</y>
       <width>151</width>
       <height>151</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Images similar to this will be retrieved</string>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="midLineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="queryResult_9">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>180</y>
       <width>151</width>
       <height>151</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Images similar to this will be retrieved</string>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="midLineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="queryResult_10">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>180</y>
       <width>151</width>
       <height>151</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Images similar to this will be retrieved</string>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="midLineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="queryResult_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>151</width>
       <height>151</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Images similar to this will be retrieved</string>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="midLineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="queryResult_12">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>340</y>
       <width>151</width>
       <height>151</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Images similar to this will be retrieved</string>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="midLineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="queryResult_13">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>340</y>
       <width>151</width>
       <height>151</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Images similar to this will be retrieved</string>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="midLineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="queryResult_14">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>340</y>
       <width>151</width>
       <height>151</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Images similar to this will be retrieved</string>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="midLineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="queryResult_15">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>340</y>
       <width>151</width>
       <height>151</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Images similar to this will be retrieved</string>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="midLineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="queryResult_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>340</y>
       <width>151</width>
       <height>151</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Images similar to this will be retrieved</string>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="midLineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>610</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Paolo D'Apice 2013</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>580</y>
      <width>411</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Recompute</string>
    </property>
    <widget class="QPushButton" name="recomputeIndexButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Index</string>
     </property>
    </widget>
    <widget class="QPushButton" name="recomputeDescriptorsButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Descriptors</string>
     </property>
    </widget>
    <widget class="QPushButton" name="recomputeVocabularyButton">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Vocabulary</string>
     </property>
    </widget>
    <widget class="QPushButton" name="recomputeQueriesButton">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>20</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Queries</string>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="queryList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>131</width>
      <height>171</height>
     </rect>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="queryImage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>131</width>
      <height>131</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Images similar to this will be retrieved</string>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid gray</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1015</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="queryTypeGroup"/>
  <buttongroup name="categoryGroup"/>
 </buttongroups>
</ui>
